src/ansys/pyensight/core/renderable.py

```diff
@@ -500,6 +500,21 @@ def update(self):
         super().update()
 
 
+# Undocumented class
+class RenderableVNCAngular(Renderable):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._rendertype = "remote"
+        self.update()
+
+    def update(self):
+        url = f"http://{self._session.hostname}:{self._session.html_port}"
```

Comment:
@mariostieriansys maybe  need to add the comment string to describe the method here?

Reply:
I avoided docstring to not generate anything in sphinx, but yes maybe the comment string is a good idea

Reply:
done, thanks!